src/controllers/orders.js

```diff
@@ -1,35 +1,153 @@
+        if(err) return res.status(500).send({message: `Error al realizar la petición: ${err}`})
+        if(!content) return res.status(404).send({ message: 'No existen ordenes asignadas a ese repartidor'})
+        res.status(200).send({content: content})
+    })
+}
+
 function getOrdersId(req, res){
     let ordersId = req.params.id_order
 
     Orders.findOne({id_order: ordersId}, (err, orders) => {
         if(err) return res.status(500).send({message: `Error al realizar la petición: ${err}`})
-        if(!orders) return res.status(404).send({message: 'No existen ordenes'})
+        if(!orders) return res.status(404).send({message: 'No la orden'})
         res.status(200).send({orders: orders})
     })
 }
 
+// Revisar
+/*async function getDataRestaurant(req, res){
+    var name_restaurant = ''
+    var rest_phone_number = 0
+    await Restaurant.findOne({id_restaurant: req.body.id_restaurant}, (err, content) => {
+        if(err) return res.status(500).send({message: `Error al realizar la petición ${err}`})
+        if(!content) return res.status(404).send({ message: 'No existe el restaurante'})
+        name_restaurant = content.restaurant_name
+        rest_phone_number = content.rest_phone_number
+    })
+    return {
+        name_restaurant, 
+        rest_phone_number
+    }
+}*/
+
+async function getDataDeliveryman(req, res){
+    var name_deliveryman = ''
+    var deliveryman_phone_number = 0
+    var deliveryman_type = ''
+    var code = ''
+    await Deliveryman.findOne({id_deliveryman: req.body.id_deliveryman}, (err, content) => {
+        if(err) return res.status(500).send({message: `Error al realizar la petición ${err}`})
+        if(!content) return res.status(404).send({message: 'El repartidor no existe'})
+        name_deliveryman = content.name + " " + content.last_name
+        deliveryman_phone_number = content.phone_number
+        deliveryman_type = content.deliveryman_type
+        code = content.code
+    })
+    return {
+        name_deliveryman,
+        deliveryman_phone_number,
+        deliveryman_type,
+        code
+    }
+}
+
+// Revisar
+/*async function getDataClient(req,res){
+    var name_client = ''
+    var client_phone_number = 0
+    var client_address = ''
+    await Client.findOne({id_client: req.body.id_client}, (err, content) => {
+        if(err) return res.status(500).send({ message: `Error al realizar la petición ${err}`})
+        if(!content) return res.status(404).send({ message: 'El cliente no existe'})
+        name_client = content.name + " " + content.last_name
+        client_phone_number = content.username
+        client_address = content.address + " #" + content.number_address
+    })
+    return {
+        name_client, 
+        client_phone_number
+    }
+}*/
+
+async function getDataCompanyThirdParty(req, res, codeDeliveryman){
+    var id_company_third_party = 0
+    await Ctp.findOne({ctp_code: codeDeliveryman}, (err, content) => {
```

Comment:
Hi! Are you trying to get only **id_tpuser**? If so, this code will help you. Also, you can delete line 121:123 and don't forget to add my next suggestion.

```suggestion
    await Ctp.findOne({ctp_code: codeDeliveryman}, ' id_tpuser' ,(err, content) => {
```

Reply:
Por el momento solamente estamos ocupando el id_tpuser pero lo deje asi por si mas adelante necesito otro campo